libiocage/lib/helpers.py

```diff
@@ -98,7 +98,8 @@ def _prettify_output(output):
 
 # helper function to validate names
 def validate_name(name):
-    validate = re.compile(r'[a-z0-9][a-z0-9\.-_]{0,31}', re.I)
+    validate = re.compile(r'[a-z0-9][a-z0-9\.-_-]{0,31}', re.I)
```

Comment:
Because without it, for example -- `11.1-RELEASE` would only match `11.1`, when we do intend to match `11.1-RELEASE` in its entirety.